test_conformance/math_brute_force/main.c

```diff
@@ -1311,218 +1252,6 @@ static int IsInRTZMode( void )
 const char *sizeNames[ VECTOR_SIZE_COUNT] = { "", "2", "3", "4", "8", "16" };
 const int  sizeValues[ VECTOR_SIZE_COUNT] = { 1, 2, 3, 4, 8, 16 };
 
-float Ulp_Error_Double( double test, long double reference )
```

Comment:
This function is rather different from the one defined in the harness. What was done on the master branch was to prefix the name with `Bruteforce_`. I suggest doing the same here to avoid potentially introducing subtle issues and make the upcoming branch merge easier.